README.md

```diff
@@ -61,6 +61,51 @@ You can also always find the full command-line options that are available with
 $ cargo fuzz --help
 ```
 
+## Generating code coverage information
+
+Use the `--coverage` option to generate precise
+[source-based code coverage](https://blog.rust-lang.org/inside-rust/2020/11/12/source-based-code-coverage.html)
+information:
+```
+$ cargo fuzz run --coverage <coverage data output file> <target>
+```
+This compiles your project using the `-Zinstrument-coverage` Rust compiler flag and generates coverage data in the
+specified file. This file can be used to generate coverage reports and visualize code-coverage information.
+If you run the fuzzer multiple times, you can specify different coverage-output file names and subsequently merge them
+into one data file.
+Read more in the [Unstable book](https://doc.rust-lang.org/beta/unstable-book/compiler-flags/source-based-code-coverage.html#installing-llvm-coverage-tools).
```

Comment:
This links to the section about installing the tools, rather than merging the coverage files (which is the next section).